docs/manual/manual_ifci_control.rst

```diff
@@ -0,0 +1,61 @@
+
+More information on setting up throttle response can be found in :ref:`manual_throttle`.
+
+The X and Y CVI convert the raw 0.0 to 1.0 values to -1.0 to 1.0. The setting 'pulsing_voltage_mode' controls whether this value is considered a percentage of supply voltage or a percentage of some maximum voltage. When set to supply voltage mode -1.0 maps to a pulsing voltage on that axis of negative battery voltage and 1.0 maps to positive battery voltage. When set to voltage limit mode you must also set the 'pulsing_voltage_limit' parameter. This will set the maximum and minimum pulsing voltage. In this mode -1.0 will map to -'pulsing_voltage_limit' and 1.0 will map to 'pulsing_voltage_limit'. Between these values the raw value is linearly mapped to the voltages inbetween.
+
+While this can work with to control a single motor module connected to a single UART port on your flight controller, you can also connect multiple motors to a single UART port for control as a bus. An example of the connection is shown below.
+
+.. figure:: ../_static/manual_images/pulsing_propeller/motor_serial_connection.png
+    :align: center
+    :alt: Bussed Serial Connection
+
+    Bussed Serial Connection
+
+This will work without changing the module IDs, but if telemetry is required you will need to set each module's module ID to a different value. This is done in the :ref:`system control client<sc_message_table>` with the 'module_id' parameter. Each motor in your bus should have different module IDs.
+
+To request telemetry with the packed control message the Telemetry ID Tail Byte (TITB) must be set to the module ID that you'd like to request telemetry from. When this is set, the corresponding motor will immediately reply with a telemetry packet. To get every motor's telemetry, the flight controller must ask each motor for telemetry individually. If you are sending control messages on an aircraft with 4 motors at 400hz then you will only receive telemetry at 100hz per motor. The telemetry data is packed in a standard IQUART message and the data structure is described in :ref:`IFCITelemetryData struct<ifcitelemetrydata_note>`.
```

Comment:
To request telemetry via the packed control message, the Telemetry ID Tail Byte (TITB) must be set to the module ID of the module whose telemetry is desired.